instrumentation/otel-webserver-module/include/core/api/opentelemetry_ngx_api.h

```diff
@@ -14,8 +14,8 @@
 * limitations under the License.
 */
 
-#ifndef APPDYNAMICS_NGX_API_H
-#define APPDYNAMICS_NGX_API_H
+#ifndef OPENTELEMETRY_NGX_API_H
```

Comment:
Aren't these nomenclatures related to core directory?

Reply:
This nomenclature changes are done for all nginx code added. Gradually, its better we do the same for all other files in core directory.